package/test_requirements.txt

```diff
@@ -4,11 +4,13 @@ kedro-telemetry>=0.1.1  # for testing telemetry integration
 bandit~=1.7
 behave~=1.2
 black~=23.3
+boto3~=1.26
 flake8~=5.0
 fastapi[all]>=0.73.0, <0.96.0
 isort~=5.11
 matplotlib~=3.5
-mypy~=0.990
+mypy==0.990
```

Comment:
I have pinned this for now due to some pydantic issues that we are resolving in another ticket.